CI: core dump errors

Example: https://circleci.com/gh/freedomofpress/securedrop-client/1163#usage-queue/containers/0


Found the cause below. Will examine more.

```
$ make test TESTS=tests/test_resources.py 
============================================================== test session starts ===============================================================
platform linux -- Python 3.5.7, pytest-3.9.3, py-1.7.0, pluggy-0.8.0 -- /home/circleci/project/.venv/bin/python
cachedir: .pytest_cache
Using --random-order-bucket=global
Using --random-order-seed=927124

rootdir: /home/circleci/project, inifile:
plugins: random-order-1.0.0, mock-1.10.0, cov-2.6.0
collected 5 items                                                                                                                                

tests/test_resources.py::test_load_css PASSED                                                                                              [ 20%]
tests/test_resources.py::test_path PASSED                                                                                                  [ 40%]
tests/test_resources.py::test_load_svg Aborted (core dumped)
make: *** [Makefile:35: test] Error 134

```

I can now replicate locally.

```
$ make test TESTS=tests/test_resources.py 
============================================================== test session starts ===============================================================
platform linux -- Python 3.5.3, pytest-3.9.3, py-1.7.0, pluggy-0.8.0 -- /home/kdas/code/securedrop-client/.venv/bin/python
cachedir: .pytest_cache
Using --random-order-bucket=global
Using --random-order-seed=204513

rootdir: /home/kdas/code/securedrop-client, inifile:
plugins: random-order-1.0.0, mock-1.10.0, cov-2.6.0
collected 5 items                                                                                                                                

tests/test_resources.py::test_load_icon Aborted
Makefile:35: recipe for target 'test' failed
make: *** [test] Error 134

```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -36,7 +36,7 @@ jobs:
     steps:
       - checkout
 
-      - run: sudo apt-get install -y sqlite3
+      - run: sudo apt-get install -y sqlite3 libqt5x11extras5
 
       - run:
           name: Install requirements and run tests

--- a/tests/test_resources.py
+++ b/tests/test_resources.py
@@ -4,6 +4,9 @@ Tests for the resources sub-module.
 import securedrop_client.resources
 from PyQt5.QtGui import QIcon, QPixmap
 from PyQt5.QtSvg import QSvgWidget
+from PyQt5.QtWidgets import QApplication
+
+app = QApplication([])
 
 
 def test_path(mocker):

